src/components/AutoCompleteSuggestions/index.tsx

```diff
@@ -26,6 +26,14 @@ function isSuggestionRenderedAbove(isEnoughSpaceAboveForBig: boolean, isEnoughSp
     return isEnoughSpaceAboveForBig || isEnoughSpaceAboveForSmall;
 }
 
+type IsEnoughSpaceAbove = Pick<MeasureParentContainerAndCursor, 'y' | 'cursorCoordinates' | 'scrollValue'> & {
+    contentHeight: number;
+    topInset: number;
+};
+function isEnoughSpaceAbove({y, cursorCoordinates, scrollValue, contentHeight, topInset}: IsEnoughSpaceAbove): boolean {
```

Comment:
I find this naming slightly confusing. If I understand what this is doing correctly, then I think something like thisi s more clear: 
```suggestion
function isEnoughSpaceToRenderMenuAboveCursor({y, cursorCoordinates, scrollValue, contentHeight, topInset}: IsEnoughSpaceAbove): boolean {
```

Reply:
👍🏻